docs/docs/cloud/concepts/data_storage_and_privacy.md

```diff
@@ -0,0 +1,121 @@
+# CLI Data Storage and Privacy
+
+This document describes how [LangGraph CLI](../../concepts/langgraph_cli.md) handles data storage, collection, and privacy. It covers what data is stored locally versus remotely, security measures, and privacy controls.
```

Comment:
We need to be clear exactly what this is covering. In this doc, you also sort of talk about the in-mem server, the local single-container server (`up`) and then sort of mention others.

Need to be up front about what this does and does not cover.